packages/salesforcedx-utils/test/jest/cli/cliCommandExecution.test.ts

```diff
@@ -0,0 +1,203 @@
+import { CancellationToken, CliCommandExecution, Command } from '../../../src';
+import {
+  CANCELATION_INTERVAL,
+  KILL_CODE,
+  NO_PID_ERROR,
+  NO_STDERR_ERROR,
+  NO_STDOUT_ERROR
+} from '../../../src/cli/cliCommandExecution';
+import { Observable } from 'rxjs/Observable';
+import * as kill from 'tree-kill';
+jest.mock('tree-kill');
```

Comment:
This mocks the module in memory in jest.  So when it's used in the code under test the underlying functions are just mocks.

Reply:
correct. If you need to mock and entire module for a test this is the easiest way to  do it.